### PR TITLE
Improves accessibility labels and hints in media library

### LINKED ISSUE #12874: Media: navigation bar buttons add, edit are not descriptive and trash button is not accessible
Steps to reproduce:

1. Go to Settings > General > Accessibility > VoiceOver and toggle the setting on.
1. In the WordPress app, go to My Site > Media.
1. Explore the buttons in the top bar for add, edit, and trash (after you go into edit mode).

**Result:** the Media navigation buttons could use better descriptions and the trash button is not accessible at all saying "Dim. Button" when it is inactive (while no media is selected) and just "Button" when it is active (while media is selected). While images are being deleted, and afterward, it's really easy to get lost with what's happening if you are navigating without sight. Here is a video with an example of deleting images while using VoiceOver: [1m12](https://cloudup.com/cBi15ozlA2Y).

> The edit button could have a hint saying how editing works (multi-select to delete), since it’s not clear what happens after activating it. 

media-edit-button media-trash-button-with-selected-images
<sup>Tested with WPiOS 13.5.0.3 (App Store version) on iPhone 6S iOS 13.2.</sup>

<p style="clear:both;"><sub>/hat tip @etoledom for the report, internal reference: p77Llu-bQ6-p2</sub></p>

### PR DESCRIPTION
**Pull request by sla8c**
Fixes [#12874](https://github.com/wordpress-mobile/WordPress-iOS/issues/12874)

To test:

1. Go to Settings > General > Accessibility > VoiceOver and toggle the setting on.
2. In the WordPress app, go to My Site > Media.
3. Explore the buttons in the top bar for add, edit, and trash (after you go into edit mode).  VoiceOver should now be more descriptive for the add and edit buttons, and the trash button should now be accessible

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
I tested in Simulator using Accessibility Inspector (I currently don't have ability to test / debug on Device)

3. What automated tests I added (or what prevented me from doing so)
None

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Media/MediaLibraryViewController.swift [hunk 1]
```diff
@@ -123,6 +123,8 @@ class MediaLibraryViewController: WPMediaPickerViewController {
             navigationItem.setLeftBarButton(UIBarButtonItem(barButtonSystemItem: .cancel, target: self, action: #selector(editTapped)), animated: false)
 
             let trashButton = UIBarButtonItem(image: .gridicon(.trash), style: .plain, target: self, action: #selector(trashTapped))
+            trashButton.accessibilityLabel = NSLocalizedString("Trash", comment: "Accessibility label for trash button in nav bar")
```
Comment (frosty) [inline WordPress/Classes/ViewRelated/Media/MediaLibraryViewController.swift]: I would probably clarify these comments slightly by saying `for trash button to delete items from the user's media library`

Comment (frosty) [reply WordPress/Classes/ViewRelated/Media/MediaLibraryViewController.swift]: Similar goes for the add buttons below.

File: WordPress/Classes/ViewRelated/Media/MediaLibraryViewController.swift [hunk 2]
```diff
@@ -132,11 +134,17 @@ class MediaLibraryViewController: WPMediaPickerViewController {
...
                 let addButton = UIBarButtonItem(barButtonSystemItem: .add, target: self, action: #selector(addTapped))
+                addButton.accessibilityLabel = NSLocalizedString("Add", comment: "Accessibility label for add button in nav bar")
+                addButton.accessibilityHint = NSLocalizedString("Add new media", comment: "Accessibility hint for add button in nav bar")
+
                 barButtonItems.append(addButton)
             }
 
             if blog.supports(.mediaDeletion) && assetCount > 0 {
                 let editButton = UIBarButtonItem(barButtonSystemItem: .edit, target: self, action: #selector(editTapped))
+                editButton.accessibilityLabel = NSLocalizedString("Edit", comment: "Accessibility label for edit button in nav bar")
```
Comment (frosty) [inline WordPress/Classes/ViewRelated/Media/MediaLibraryViewController.swift]: The comment here could mention `edit button to enable multi selection mode in the user's media library` or something similar.

**Review by frosty — COMMENTED** [review]
Thanks for fixing this! I tested on device and it works as expected. Just added a few suggestions for clarifying the localization comments slightly.

File: WordPress/Classes/ViewRelated/Media/MediaLibraryViewController.swift (hunk 1 above)
Comment (sla8c) [reply WordPress/Classes/ViewRelated/Media/MediaLibraryViewController.swift]: Np done!

File: WordPress/Classes/ViewRelated/Media/MediaLibraryViewController.swift (hunk 2 above)
Comment (sla8c) [reply WordPress/Classes/ViewRelated/Media/MediaLibraryViewController.swift]: No prob - done

**Review by frosty — APPROVED** [review]
Looks good! I just restarted the failed buildkite runs – once everything's green, let's merge

**Pull request merged**